tests/contrib/anthropic/test_anthropic_llmobs.py

```diff
@@ -14,7 +14,14 @@
 parameter. The user has provided the location of "San Francisco, CA" in their question, so we have \
 the necessary information to make the API call.\n\nNo other tools are needed to answer this question. \
 We can proceed with calling the get_weather tool with the provided location.\n</thinking>'
-WEATHER_OUTPUT_MESSAGE_2 = '[tool: get_weather]\n\n{"location": "San Francisco, CA"}'
+WEATHER_OUTPUT_MESSAGE_2 = [
```

Comment:
Could we rename this to `WEATHER_OUTPUT_TOOL_CALL` instead to be more accurate? Thanks!